pubspec.lock

```diff
@@ -266,21 +266,20 @@ packages:
     source: hosted
     version: "2.9.4"
   carp_mobile_sensing:
-    dependency: "direct main"
+    dependency: "direct overridden"
     description:
-      name: carp_mobile_sensing
-      sha256: "8d371939d46e91bd96141fc558a1f35f64d81e3555176250efd39a25d0ca1d19"
-      url: "https://pub.dev"
-    source: hosted
-    version: "1.7.2"
+      path: "../carp/carp.sensing-flutter/carp_mobile_sensing"
+      relative: true
+    source: path
+    version: "1.8.1"
```

Comment:
This needs to be fixed before merge

Reply:
This should not be an issues. The pubspec.lock file is generated from the pubspec file when the app is build. Actually, there has been some discussion previously why the lock file isn't gitignored in the first place. But Lars found some recommendation somewhere that lock files should be committed for APPS whereas it should be ignored for PACKAGES.

Reply:
... but I've committed the lock file now, and it should be pointing to pub.dev